addons/crm/wizard/base_partner_merge.py

```diff
@@ -282,9 +289,14 @@ def _merge(self, partner_ids, dst_partner=None):
         _logger.info("dst_partner: %s", dst_partner.id)
 
         # FIXME: is it still required to make and exception for account.move.line since accounting v9.0 ?
```

Comment:
I would be interested in discussing this `FIXME` question... Is it a problem to merge partners that have move lines technically? Or is this just there for the goal of preventing users from messing up their accounting?

If it's a functional issue, I'd rather we improve the wizard further (possibly in master since it's not *urgent* and could be a good newbie task) so that e.g. accounting advisors can overrule this decision from the UI (and not just code as we're doing here).